File: split/initspan.pas
procedure initspan(p: halfword);
begin
  pushnest;
  if curlist.modefield = -102 then
    curlist.auxfield.hh.lh := 1000
  else
  begin
    curlist.auxfield.int := -65536000;
    normalparagraph;
  end;
  curspan := p;
end;

File: cpp/initspan.cpp
#include "initspan.h"
#include "pushnest.h"
#include "normalparagraph.h"

void initspan(halfword p)
{
	pushnest();
	if (curlist.modefield == -102)
		curlist.auxfield.hh.lh = 1000;
	else
	{
		curlist.auxfield.int_ = -65536000;
		normalparagraph();
	}
	curspan = p;
}
